src/client/index.ts

```diff
@@ -19,26 +103,82 @@ export function removeEventListener<E extends NotificationEvent>(eventType: E['t
     eventEmitter.removeListener(eventType, listener);
 }
 
+/**
+ * Creates a new Notification.
+ *
+ * The Notification will appear in the Notification Center or as a toast if the Center is not visible.
```

Comment:
I think we should be clear that it will always appear in the centre, and also as a toast if the centre is hidden when raised